cfa/index.markdown

```diff
@@ -0,0 +1,211 @@
+*   Not explaining how to interpret results. For instance, if an artifact ran successfully and produced the output that was the basis for the article, but without any way for reviewers to check the output for consistency with the article. Examples included generating a list of warnings without documenting which were true vs. false positives and generating large tables of numbers that are presented graphically in the article without providing a way to generate analogous visualizations.  
+
+
+### COI
+
+Conflicts of interests for AEC members are handled by the chairs. Conflicts of interest involving one of the two AEC chairs are handled by the other AEC chair, or the associate editor of the current volume if both chairs are conflicted.
+
+### FAQ
+
+This list will be updated with useful questions as time goes on.
+
+**My artifact requires hundreds of GB of RAM / hundreds of CPU hours / a specialized GPU / etc., that the AEC members may not have access to. How can we submit an artifact?**
+
+If the tool can run on an average modern machine, but may run extremely slow in comparison to the hardware used for the article's evaluation, please document the expected running time on your own hardware, and point to examples the AEC may be able to replicate in less time. If your system will simply not work at all without hundreds of GB of RAM, or other hardware requirements that most typical graduate student machines will not satisfy, please _contact the AEC chairs in advance to make arrangements_. This can include options such as the authors paying for a cloud instance with the required hardware, which reviewers can have _anonymous_ access to (the AEC chairs play proxy to communicate when the instance may be off to save the authors money). Submissions using cloud instances or similar that are not cleared with the AEC Chairs in advance will be summarily rejected.
+
+**Can my artifact be accepted if some of the article’s claims are not supported by the artifact, for example, if some benchmarks are omitted or the artifact does not include tools we experimentally compare against in the article?**
+
+In general yes (if good explanations are provided, as explained above), but if such claims are essential to the overall results of the article, the artifact will not receive the Functional or Reusable badge. As an extreme example, an artifact consisting of a working tool submitted with no benchmarks (e.g., if all benchmarks have sources that may not be redistributed) would not receive the functional or reusable badge. Note that the artifact can still receive the Available badge.
+
+**Why do we need to use Zenodo for the _Available_ badge? Why not GitHub?**
+
+Commercial repositories are unreliable in that there is no guarantee the evaluated artifact will remain available indefinitely. Users can delete public repositories or their accounts. And in addition to universities deleting departmental URLs over time, hosting companies also sometimes simply delete data.
```

Comment:
Maybe add that zenodo can automagically create a stable entity from a github release. This should aleviate the "why not gh" issue